Files/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -56,60 +1065,61 @@
-                                    <Grid
+                ToolTipService.ToolTip="{x:Bind ItemName, Mode=OneWay}" Loaded="Grid_Loaded">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="24" />
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="Auto" />
+                        </Grid.ColumnDefinitions>
+                        <Grid Height="16" CornerRadius="2" Margin="-10,0" x:Name="SelectedFolderIndicator" Width="3" Background="Gray" Opacity="0" HorizontalAlignment="Left"></Grid>
+                        <Grid
                     Grid.Column="0"
                     Width="24"
                     Height="24"
                     Opacity="{x:Bind Opacity, Mode=OneWay}"
                     Tag="ItemImage">
-                                        <Image
+                            <Image
                         x:Name="Picture"
                         HorizontalAlignment="Stretch"
                         VerticalAlignment="Stretch"
-                        x:Load="{x:Bind LoadFileIcon, Mode=OneWay}"
+                        
+                                            Visibility="{x:Bind LoadFileIcon, Mode=OneWay}"
                         x:Phase="1"
                         Source="{x:Bind FileImage, Mode=OneWay}"
                         Stretch="Uniform" />
-                                        <Image
+                            <Image
                         x:Name="FolderGlyph"
                         HorizontalAlignment="Stretch"
                         VerticalAlignment="Stretch"
-                        x:Load="{x:Bind LoadFolderGlyph, Mode=OneWay}"
+                        Visibility="{x:Bind LoadFolderGlyph, Mode=OneWay}"
                         x:Phase="1"
                         Stretch="Uniform">
-                                            <Image.Source>
-                                                <SvgImageSource
+                                <Image.Source>
+                                    <SvgImageSource
                                 RasterizePixelHeight="256"
                                 RasterizePixelWidth="256"
                                 UriSource="{x:Bind FolderIconSourceLarge}" />
-                                            </Image.Source>
-                                        </Image>
-                                        <Viewbox
+                                </Image.Source>
+                            </Image>
+                            <Viewbox
                         x:Name="TypeUnknownGlyph"
                         MaxWidth="24"
                         MaxHeight="24"
                         HorizontalAlignment="Stretch"
                         VerticalAlignment="Stretch"
-                        x:Load="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"
+                        Visibility="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"
```

Comment:
Can you continue using x:Load?

Reply:
Could you change the name to UnknownTypeGlyphVisibility then @TyJOrtiz

Reply:
There is no `UnknownTypeGlyphVisibility ` in ListedItem

Reply:
Is LoadUnknownTypeGlyph a boolean? Then it won't work with Visibility property

Reply:
Windows 10 has supported BoolToVisibiltiyConverter natively since 15063